app/authProvider.js

```diff
@@ -55,13 +56,32 @@ export class AuthProvider {
     app.use(this.passport.session());
   }
 
-  authenticate(options) {
+  login() {
     return (req, res, next) => {
+      const options = {
+        successReturnToOrRedirect: '/',
```

Comment:
Do we need this now? can you try it without. Since we are saying in the state where we want to redirect back to right?

Reply:
nope - deleted